### DISCUSSION TITLE
是否支持GPT-4o mini？

### DISCUSSION THREAD

**Discussion by friedpine** [BurnHair/docs#166] [功能建议]
如果支持的话，大概什么时候可以用上？

**Comment by xiaolinbaba** [comment]
老大，什么时候支持gpt-4o-mini

**Reply by FeliksBak** [reply]
已经支持了

**Reply by xiaolinbaba** [reply]
![image](https://github.com/user-attachments/assets/4cef7fe8-03de-414a-8fb5-068ac86809d9)
没有哦

**Reply by FeliksBak** [reply]
> ![image](https://private-user-images.githubusercontent.com/12092772/350878099-4cef7fe8-03de-414a-8fb5-068ac86809d9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE2NTI1OTMsIm5iZiI6MTcyMTY1MjI5MywicGF0aCI6Ii8xMjA5Mjc3Mi8zNTA4NzgwOTktNGNlZjdmZTgtMDNkZS00MTRhLThmYjUtMDY4YWM4NjgwOWQ5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzIyVDEyNDQ1M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWUwY2JhMmVhZGVjNDE1NjI0ODMzMWFiZjg2NTc5YTI5MWEyY2ZlOWU2YTEyYjVhOGI5ZGZkNGVmZDQ2NjhhNjMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.0JHMi_HvXcYam89U0bZPfPMxJ9vhTNlIFseclY34cAM) 没有哦

有没有一种可能，“gpt-4o-mini”才是常用的模型名

**Reply by xiaolinbaba** [reply]
是我大意了，谢谢老铁，/抱拳/抱拳

**Comment by bxb100** [comment]
@BennyThink `https://burn.hair/v1/models` 接口没更新 = .=

**Reply by BennyThink** [reply]
有了！

**Reply by bxb100** [reply]
🎉(>ω<)🎉

**Comment by Lion066** [comment]
`{
  "error": {
    "message": "当前分组 vip 下对于模型 gpt-4o-mini 无可用渠道 (request id: 20240731074716292072911ZJ1fdbgs)",
    "type": "one_api_error"
  },
  "endpoint": "https://***.hair/v1",
  "provider": "openai"
}`
昨天2024-07-30还可以用，今天2024-07-31怎么不能用了？

**Reply by BennyThink** [reply]
出了点问题，会修的稍等

**Reply by deanzz** [reply]
大佬加油💪🏻

**Comment by BennyThink** [comment]
官宣支持！
